drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -2220,6 +2224,7 @@ virtio_vdpa_dev_probe(struct rte_pci_driver *pci_drv __rte_unused,
 	}
 
 	priv->iommu_domain = iommu_domain;
+	pthread_mutex_init(&priv->domain_lock, NULL);
```

Comment:
if 2 vf use same iommu domain, but they will use different lock to protect same iommu domain?

Reply:
yes, current code is wrong